IFC4x3/Properties/s/StrandedStructure_0R$627I2516eOH2rRvhsTB/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StrandedStructure_0R627I2516eOH2rRvhsTB" Name="StrandedStructure" UniqueId="1bfc6087-4821-411a-8611-0b56f9af674b" PrimaryDataType="IfcReal" />
```

Comment:
This property doesn't make sense to me, a real?  Documentation might need improvement.

Reply:
Agree with you. No need of this property, Stranded Type is enought. commit : 305806220fb93c9010687f137214b3f48df310cb